gulp-tasks/publish-glitch.js

```diff
@@ -0,0 +1,62 @@
+/*
+  Copyright 2018 Google LLC
+
+  Use of this source code is governed by an MIT-style
+  license that can be found in the LICENSE file or at
+  https://opensource.org/licenses/MIT.
+*/
+
+const gulp = require('gulp');
+// const del = require('del');
+const git = require('gulp-git');
```

Comment:
I don't think you should use `gulp-git` here. It is probably making you more tied to the `gulp` way of doing things than you want to be.

There are also some helper libraries like `tempy` and `fs-extra` that we use elsewhere and could help here.

There's an example at https://github.com/GoogleChrome/workbox/commit/89840b22da92a76b09da0d69387160e4f2888255 of just spawning a local process using `async`/`await` (in that case, `yarn`, but you should be able to do it with `git` as well.)

```
const childProcess = require('child_process');
const fse = require('fs-extra');
const tempy = require('tempy');
const util = require('util');

const exec = util.promisify(childProcess.exec);
```